### DISCUSSION TITLE
URL field clickable in List?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#14385] [Archive]
How can I have a column of URLs in a list?
The current behavior is to skip tags in fields when showing in a list and to have the click opening the detail row.

**Comment by rijkvanzanten** [comment]
There's currently no "Link" display yet, though there's an open suggestion about that here https://github.com/directus/directus/pull/8520

**Comment by Pachat** [comment]
Thanks to Directus versatility I could achieve this with a simple Display component
display.vue
```
<template>
	<a :href="value" target="_blank">🔗</a>
</template>

<script>
export default {
	props: {
		value: {
			type: String,
			default: null,
		},
	},
};
</script>
```

When clicking on the link in the list

1. the new page is opened
2. the list opens the detail which is exactly what is wanted.
